### PR TITLE
Updated docs for labels rollout

### PR DESCRIPTION
**Pull request by Dorthu**

### CODE REVIEW

File: _includes/reference/objects-intro.html [hunk 1]
```diff
@@ -10,15 +10,9 @@
...
-            the form of <code>prefix_####</code>. The prefix identifies the
-            type of record you're dealing with, and #### is a unique integer ID.
-        </p>
-        <p>
-            Many of the objects the API returns have relationships to other objects. For
-            convenience, we will often return a shorter version of the related object's
-            record with a few properties included, and the ID. You can always do another
-            request to get the full record for any related objects if necessary.
+            objects you can manage with it. These objects will always include an identifier,
+            which will be a string used to refernece that object.  These identifiers,
```
Comment (eatonphil) [inline _includes/reference/objects-intro.html]: refernece

File: _includes/reference/objects-intro.html [hunk 2]
```diff
@@ -10,15 +10,9 @@
...
+            called "labels," are returned in the "id" attribute.
```
Comment (eatonphil) [inline _includes/reference/objects-intro.html]: "labels",

Comment (Dorthu) [inline _includes/reference/objects-intro.html]: normal grammar says the comma should go inside the quotes, but I feel like technical talk like this is a grey area..

Comment (eatonphil) [inline _includes/reference/objects-intro.html]: Interesting. This random [site](http://www.grammarbook.com/punctuation/quotes.asp) (rule 8a) agrees with you!

**Comment by jamesottinger** [comment]
👍

**Pull request merged**